src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/DynamicInterfaceCastableImplementation.cs

```diff
@@ -99,7 +99,8 @@ private static void AnalyzeType(SymbolAnalysisContext context, INamedTypeSymbol
             {
                 foreach (var member in iface.GetMembers())
                 {
-                    if (!member.IsStatic
+                    if (member.IsAbstract &&
+                        member.Kind != SymbolKind.NamedType
```

Comment:
Nit: to match the other operator placement

```suggestion
                    if (member.IsAbstract
                        && member.Kind != SymbolKind.NamedType
```